urdf/scripts/mesh_distortion/README.md

```diff
@@ -33,22 +33,30 @@ There are two Coke can models as examples:
 `models/dave_object_models/models/Coke_Can/meshes/coke_can.obj`.
 Choose Transform while importing: Z Up, X Forward.
 
+Tip: To view the textured object in the viewport, click on the globe icon in the
```

Comment:
Minor suggestion,
We need not only the `obj` file but also `mtl` and  `../materials/coke.png` to load the texture correctly.

Reply:
I added a line 5f7030b
Those files are considered part of the object model files, and it's usually assumed that the users have the entire `Coke` or `Coke_can` folder, which would include all the associated files.

Side note: I noticed that you copied some files from `Coke_Can` to a flat directory in the `nps_uw_multibeam_sonar` package, but you should really copy the entire folder, because we use relative paths inside the 3D model files to reference to the `materials`, `scripts`, etc. subfolders. Those folders also contain `model.sdf` and `model.config`, which Gazebo looks for (there's an error about missing `'model.config` for `nps_uw_multibeam_sonar/models/Coke_Can`). All models on Ignition Fuel follow this directory structure.

I would also recommend renaming the copy in `nps_uw_multibeam_sonar`, because if I do catkin install, there are two models of the same name (Coke_Can) in the GAZEBO_MODEL_PATH, and Gazebo picks one of them, such that even when I do `roslaunch dave_demo_launch distorted_coke.launch`, it picks the one in `nps_uw_multibeam_sonar` instead of the one in `dave_object_models` and gives that `model.config` error. Copying that entire folder, and renaming the folder in `nps_uw_multibeam_sonar` should disambiguate between them.

Reply:
Gotcha